core/templates/pages/signup-page/signup-page.component.ts

```diff
@@ -125,7 +125,7 @@ angular.module('oppia').component('signupPage', {
         }
 
         var defaultDashboard = DASHBOARD_TYPE_LEARNER;
-        var returnUrl = window.decodeURIComponent(
+        var returnUrl = decodeURIComponent(
```

Comment:
Did you test all these changes manually?

Reply:
Sorry I didn't understand what you meant

Reply:
I mean after doing the changes, did you try visiting the signup page and trying it?

Reply:
Yes, I did, it's working perfectly!